gadget/install/install.go

```diff
@@ -324,6 +339,24 @@ func onDiskStructsSortedIdx(vss map[int]*gadget.OnDiskStructure) []int {
 	return yamlIdxSl
 }
 
+func determineDeviceVolumes(gi *gadget.Info) (map[string]*gadget.Volume, error) {
+	if len(gi.VolumeAssignments) != 0 {
+		devVols, err := gadget.VolumesForCurrentDeviceAssignment(gi)
```

Comment:
what if we changed the signature of the function a little such that we could have:
```suggestion
		vols, explicitlyAssigned, err := gadget.VolumesForCurrentDevice(gi)
		if explicitlyAssigned {
			// log
		}
```
were (vols, explicitlyAssigned) is (assigned volumes, true) if there was a match, otherwise it's simply (gi.Volumes, false) ?

Reply:
I did something similar to this